tools/distrib/fix_build_deps.py

```diff
@@ -498,7 +507,10 @@ def best(self, scorer):
             choices = new_choices
 
         best = None
-        final_scorer = lambda x: (total_avoidness(x), scorer(x), total_score(x))
+
+        def final_scorer(x):
```

Comment:
Why can't this be a lambda?

Reply:
blame yapf :P

Reply:
It was actually my vscode formatter. Reverted.